pkg/tuned/controller.go

```diff
@@ -80,7 +80,7 @@ const (
 	// be generous and give it 10s.
 	tunedGracefulExitWait = time.Second * time.Duration(10)
 	ocpTunedHome          = "/var/lib/ocp-tuned"
-	ocpTunedRunDir        = "/run/" + programName
+	ocpTunedRunDir        = "/run/ocp-tuned"
```

Comment:
Thank you for doing this.  However, I believe we need a bit more changes.  We can keep the previous version `= /run/" + programName`, but other changes will be necessary.  This is what I have in one of my cleanup PRs, perhaps you find it useful (not tested with your code):

```
diff --git a/assets/performanceprofile/configs/ocp-tuned-one-shot.service b/assets/performanceprofile/configs/ocp-tuned-one-shot.service
index d0ad4b71..60f39e36 100644
--- a/assets/performanceprofile/configs/ocp-tuned-one-shot.service
+++ b/assets/performanceprofile/configs/ocp-tuned-one-shot.service
@@ -36,7 +36,7 @@ ExecStart=/usr/bin/podman run \
     --volume /run/tuned:/run/tuned:rslave \
     --volume /run/systemd:/run/systemd:rslave \
     --volume /sys:/sys:rslave \
-    --entrypoint '["/usr/bin/cluster-node-tuning-operator","openshift-tuned","--in-cluster=false","--one-shot=true","-v=1"]' \
+    --entrypoint '["/usr/bin/cluster-node-tuning-operator","ocp-tuned","--in-cluster=false","--one-shot=true","-v=1"]' \
     $NTO_IMAGE
 Environment=PODMAN_SYSTEMD_UNIT=%n
 EnvironmentFile=-/var/lib/ocp-tuned/image.env
diff --git a/assets/tuned/manifests/ds-tuned.yaml b/assets/tuned/manifests/ds-tuned.yaml
index 13bb57e5..c94842f7 100644
--- a/assets/tuned/manifests/ds-tuned.yaml
+++ b/assets/tuned/manifests/ds-tuned.yaml
@@ -23,7 +23,7 @@ spec:
     spec:
       serviceAccountName: tuned
       containers:
-      - command: ["/usr/bin/cluster-node-tuning-operator","openshift-tuned","--in-cluster","-v=0"]
+      - command: ["/usr/bin/cluster-node-tuning-operator","ocp-tuned","--in-cluster","-v=0"]
         resources:
           requests:
             cpu: 10m
diff --git a/pkg/apis/tuned/v1/tuned_types.go b/pkg/apis/tuned/v1/tuned_types.go
index b0dde38c..7dc303e0 100644
--- a/pkg/apis/tuned/v1/tuned_types.go
+++ b/pkg/apis/tuned/v1/tuned_types.go
@@ -20,7 +20,7 @@ const (
 	TunedClusterOperatorResourceName = "node-tuning"
 
 	// Name of the NTO operand for versioning in ClusterOperator.
-	TunedOperandName = "openshift-tuned"
+	TunedOperandName = "ocp-tuned"
 
 	// TunedBootcmdlineAnnotationKey is a Node-specific annotation denoting kernel command-line parameters
 	// calculated by TuneD for the current profile applied to that Node.
diff --git a/test/e2e/performanceprofile/testdata/render-expected-output/bootstrap/extra-ctrcfg/openshift-bootstrap-master_machineconfig.yaml b/test/e2e/performanceprofile/testdata/render-expected-output/bootstrap/extra-ctrcfg/openshift-bootstrap-master_machineconfig.yaml
index ed168e7b..8cd628b7 100644
--- a/test/e2e/performanceprofile/testdata/render-expected-output/bootstrap/extra-ctrcfg/openshift-bootstrap-master_machineconfig.yaml
+++ b/test/e2e/performanceprofile/testdata/render-expected-output/bootstrap/extra-ctrcfg/openshift-bootstrap-master_machineconfig.yaml
@@ -204,7 +204,7 @@ spec:
               --volume /run/tuned:/run/tuned:rslave \
               --volume /run/systemd:/run/systemd:rslave \
               --volume /sys:/sys:rslave \
-              --entrypoint '["/usr/bin/cluster-node-tuning-operator","openshift-tuned","--in-cluster=false","--one-shot=true","-v=1"]' \
+              --entrypoint '["/usr/bin/cluster-node-tuning-operator","ocp-tuned","--in-cluster=false","--one-shot=true","-v=1"]' \
               $NTO_IMAGE
           Environment=PODMAN_SYSTEMD_UNIT=%n
           EnvironmentFile=-/var/lib/ocp-tuned/image.env
diff --git a/test/e2e/performanceprofile/testdata/render-expected-output/bootstrap/extra-ctrcfg/openshift-bootstrap-worker_machineconfig.yaml b/test/e2e/performanceprofile/testdata/render-expected-output/bootstrap/extra-ctrcfg/openshift-bootstrap-worker_machineconfig.yaml
index 5e3339ee..3c50f947 100644
--- a/test/e2e/performanceprofile/testdata/render-expected-output/bootstrap/extra-ctrcfg/openshift-bootstrap-worker_machineconfig.yaml
+++ b/test/e2e/performanceprofile/testdata/render-expected-output/bootstrap/extra-ctrcfg/openshift-bootstrap-worker_machineconfig.yaml
@@ -204,7 +204,7 @@ spec:
               --volume /run/tuned:/run/tuned:rslave \
               --volume /run/systemd:/run/systemd:rslave \
               --volume /sys:/sys:rslave \
-              --entrypoint '["/usr/bin/cluster-node-tuning-operator","openshift-tuned","--in-cluster=false","--one-shot=true","-v=1"]' \
+              --entrypoint '["/usr/bin/cluster-node-tuning-operator","ocp-tuned","--in-cluster=false","--one-shot=true","-v=1"]' \
               $NTO_IMAGE
           Environment=PODMAN_SYSTEMD_UNIT=%n
           EnvironmentFile=-/var/lib/ocp-tuned/image.env
diff --git a/test/e2e/performanceprofile/testdata/render-expected-output/bootstrap/extra-mcp/openshift-bootstrap-master_machineconfig.yaml b/test/e2e/performanceprofile/testdata/render-expected-output/bootstrap/extra-mcp/openshift-bootstrap-master_machineconfig.yaml
index 0fed5d04..cca7c9ee 100644
--- a/test/e2e/performanceprofile/testdata/render-expected-output/bootstrap/extra-mcp/openshift-bootstrap-master_machineconfig.yaml
+++ b/test/e2e/performanceprofile/testdata/render-expected-output/bootstrap/extra-mcp/openshift-bootstrap-master_machineconfig.yaml
@@ -204,7 +204,7 @@ spec:
               --volume /run/tuned:/run/tuned:rslave \
               --volume /run/systemd:/run/systemd:rslave \
               --volume /sys:/sys:rslave \
-              --entrypoint '["/usr/bin/cluster-node-tuning-operator","openshift-tuned","--in-cluster=false","--one-shot=true","-v=1"]' \
+              --entrypoint '["/usr/bin/cluster-node-tuning-operator","ocp-tuned","--in-cluster=false","--one-shot=true","-v=1"]' \
               $NTO_IMAGE
           Environment=PODMAN_SYSTEMD_UNIT=%n
           EnvironmentFile=-/var/lib/ocp-tuned/image.env
diff --git a/test/e2e/performanceprofile/testdata/render-expected-output/bootstrap/extra-mcp/openshift-bootstrap-worker_machineconfig.yaml b/test/e2e/performanceprofile/testdata/render-expected-output/bootstrap/extra-mcp/openshift-bootstrap-worker_machineconfig.yaml
index 5e3339ee..3c50f947 100644
--- a/test/e2e/performanceprofile/testdata/render-expected-output/bootstrap/extra-mcp/openshift-bootstrap-worker_machineconfig.yaml
+++ b/test/e2e/performanceprofile/testdata/render-expected-output/bootstrap/extra-mcp/openshift-bootstrap-worker_machineconfig.yaml
@@ -204,7 +204,7 @@ spec:
               --volume /run/tuned:/run/tuned:rslave \
               --volume /run/systemd:/run/systemd:rslave \
               --volume /sys:/sys:rslave \
-              --entrypoint '["/usr/bin/cluster-node-tuning-operator","openshift-tuned","--in-cluster=false","--one-shot=true","-v=1"]' \
+              --entrypoint '["/usr/bin/cluster-node-tuning-operator","ocp-tuned","--in-cluster=false","--one-shot=true","-v=1"]' \
               $NTO_IMAGE
           Environment=PODMAN_SYSTEMD_UNIT=%n
           EnvironmentFile=-/var/lib/ocp-tuned/image.env
diff --git a/test/e2e/performanceprofile/testdata/render-expected-output/bootstrap/no-mcp/openshift-bootstrap-master_machineconfig.yaml b/test/e2e/performanceprofile/testdata/render-expected-output/bootstrap/no-mcp/openshift-bootstrap-master_machineconfig.yaml
index 0fed5d04..cca7c9ee 100644
--- a/test/e2e/performanceprofile/testdata/render-expected-output/bootstrap/no-mcp/openshift-bootstrap-master_machineconfig.yaml
+++ b/test/e2e/performanceprofile/testdata/render-expected-output/bootstrap/no-mcp/openshift-bootstrap-master_machineconfig.yaml
@@ -204,7 +204,7 @@ spec:
               --volume /run/tuned:/run/tuned:rslave \
               --volume /run/systemd:/run/systemd:rslave \
               --volume /sys:/sys:rslave \
-              --entrypoint '["/usr/bin/cluster-node-tuning-operator","openshift-tuned","--in-cluster=false","--one-shot=true","-v=1"]' \
+              --entrypoint '["/usr/bin/cluster-node-tuning-operator","ocp-tuned","--in-cluster=false","--one-shot=true","-v=1"]' \
               $NTO_IMAGE
           Environment=PODMAN_SYSTEMD_UNIT=%n
           EnvironmentFile=-/var/lib/ocp-tuned/image.env
diff --git a/test/e2e/performanceprofile/testdata/render-expected-output/bootstrap/no-mcp/openshift-bootstrap-worker_machineconfig.yaml b/test/e2e/performanceprofile/testdata/render-expected-output/bootstrap/no-mcp/openshift-bootstrap-worker_machineconfig.yaml
index 5e3339ee..3c50f947 100644
--- a/test/e2e/performanceprofile/testdata/render-expected-output/bootstrap/no-mcp/openshift-bootstrap-worker_machineconfig.yaml
+++ b/test/e2e/performanceprofile/testdata/render-expected-output/bootstrap/no-mcp/openshift-bootstrap-worker_machineconfig.yaml
@@ -204,7 +204,7 @@ spec:
               --volume /run/tuned:/run/tuned:rslave \
               --volume /run/systemd:/run/systemd:rslave \
               --volume /sys:/sys:rslave \
-              --entrypoint '["/usr/bin/cluster-node-tuning-operator","openshift-tuned","--in-cluster=false","--one-shot=true","-v=1"]' \
+              --entrypoint '["/usr/bin/cluster-node-tuning-operator","ocp-tuned","--in-cluster=false","--one-shot=true","-v=1"]' \
               $NTO_IMAGE
           Environment=PODMAN_SYSTEMD_UNIT=%n
           EnvironmentFile=-/var/lib/ocp-tuned/image.env
diff --git a/test/e2e/performanceprofile/testdata/render-expected-output/default/cpuFrequency/manual_machineconfig.yaml b/test/e2e/performanceprofile/testdata/render-expected-output/default/cpuFrequency/manual_machineconfig.yaml
index d461b98d..7234cb89 100644
--- a/test/e2e/performanceprofile/testdata/render-expected-output/default/cpuFrequency/manual_machineconfig.yaml
+++ b/test/e2e/performanceprofile/testdata/render-expected-output/default/cpuFrequency/manual_machineconfig.yaml
@@ -206,7 +206,7 @@ spec:
               --volume /run/tuned:/run/tuned:rslave \
               --volume /run/systemd:/run/systemd:rslave \
               --volume /sys:/sys:rslave \
-              --entrypoint '["/usr/bin/cluster-node-tuning-operator","openshift-tuned","--in-cluster=false","--one-shot=true","-v=1"]' \
+              --entrypoint '["/usr/bin/cluster-node-tuning-operator","ocp-tuned","--in-cluster=false","--one-shot=true","-v=1"]' \
               $NTO_IMAGE
           Environment=PODMAN_SYSTEMD_UNIT=%n
           EnvironmentFile=-/var/lib/ocp-tuned/image.env
diff --git a/test/e2e/performanceprofile/testdata/render-expected-output/default/manual_machineconfig.yaml b/test/e2e/performanceprofile/testdata/render-expected-output/default/manual_machineconfig.yaml
index fa3493f8..f5863b49 100644
--- a/test/e2e/performanceprofile/testdata/render-expected-output/default/manual_machineconfig.yaml
+++ b/test/e2e/performanceprofile/testdata/render-expected-output/default/manual_machineconfig.yaml
@@ -222,7 +222,7 @@ spec:
               --volume /run/tuned:/run/tuned:rslave \
               --volume /run/systemd:/run/systemd:rslave \
               --volume /sys:/sys:rslave \
-              --entrypoint '["/usr/bin/cluster-node-tuning-operator","openshift-tuned","--in-cluster=false","--one-shot=true","-v=1"]' \
+              --entrypoint '["/usr/bin/cluster-node-tuning-operator","ocp-tuned","--in-cluster=false","--one-shot=true","-v=1"]' \
               $NTO_IMAGE
           Environment=PODMAN_SYSTEMD_UNIT=%n
           EnvironmentFile=-/var/lib/ocp-tuned/image.env
diff --git a/test/e2e/performanceprofile/testdata/render-expected-output/no-ref/manual_machineconfig.yaml b/test/e2e/performanceprofile/testdata/render-expected-output/no-ref/manual_machineconfig.yaml
index 5854d493..74baace2 100644
--- a/test/e2e/performanceprofile/testdata/render-expected-output/no-ref/manual_machineconfig.yaml
+++ b/test/e2e/performanceprofile/testdata/render-expected-output/no-ref/manual_machineconfig.yaml
@@ -221,7 +221,7 @@ spec:
               --volume /run/tuned:/run/tuned:rslave \
               --volume /run/systemd:/run/systemd:rslave \
               --volume /sys:/sys:rslave \
-              --entrypoint '["/usr/bin/cluster-node-tuning-operator","openshift-tuned","--in-cluster=false","--one-shot=true","-v=1"]' \
+              --entrypoint '["/usr/bin/cluster-node-tuning-operator","ocp-tuned","--in-cluster=false","--one-shot=true","-v=1"]' \
               $NTO_IMAGE
           Environment=PODMAN_SYSTEMD_UNIT=%n
           EnvironmentFile=-/var/lib/ocp-tuned/image.env
diff --git a/version/version.go b/version/version.go
index fee2fd57..7327de09 100644
--- a/version/version.go
+++ b/version/version.go
@@ -1,7 +1,7 @@
 package version
 
 const (
-	OperandFilename          = "openshift-tuned"
+	OperandFilename          = "ocp-tuned"
 	OperatorFilename         = "cluster-node-tuning-operator"
 	ReleaseVersionEnvVarName = "RELEASE_VERSION"
 )

```

Reply:
In theory I have no problems merging cleanups and rebasing on them, or incorporating in this PR. In practice, this specific PR is quite big and old, and got (correctly!) preempted multiple times. The risk of this PR starving out is becoming concrete, so I think we need to take into account and figure out a plan forward.
From my side there are test failures caused by latest changes which I'm prioritizing so the CI becomes green again.

Reply:
> From my side there are test failures caused by latest changes which I'm prioritizing so the CI becomes green again.

Absolutely, let's get this cleaned up and merged. Not sure we can get this done this week though, I'm out tomorrow/Friday.

Reply:
Ack, I'll make an effort to have existing open issue solved by monday.